src/SupervisorOptions.php

```diff
@@ -160,6 +167,7 @@ class SupervisorOptions
      * @param  int  $memory
      * @param  int  $timeout
      * @param  int  $sleep
+     * @param  int  $rest
```

Comment:
Adding the new option in the middle of the construct is a major breaking change. Please move it to the end.

Reply:
Right, sorry - still getting the hang of this and I always forget to consider breaking changes. This is done.